.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true,
+        "mocha": true
```

Comment:
We are using `jest` instead of `mocha`. Let it be `jest: true`.
You can also leverage some of the configs [here](https://gist.github.com/sa-adebayo/fe846cfb1c8c16660a7343aa94d21d65)